docs/modules/configuration/pages/dynamic-config.adoc

```diff
@@ -58,7 +58,7 @@ Although you can configure members to xref:security:native-client-security.adoc#
 
 Dynamic configuration is supported for the following data structures:
 
-- Map
+- Map (apart from xref:tiered-storage:overview.adoc[Tiered Storage])
```

Comment:
I think there's a misunderstanding. What you can't do is to turn on Tiered Storage on an existing map. But I believe it's still possible to dynamically add a new map that has TS enabled. Am I wrong @petrpleshachkov ?

Reply:
Yes, I meant to turn into tstore existing map is not possible.

Reply:
@JakeSCahill Therefore, I propose to remove this docs change completely.

Reply:
@Holmistr Wouldn't it be better to change it to 'You cannot add Tiered Storage configuration to an existing map'?

Reply:
Well, since we don't actually support any dynamic configuration changes in existing maps (we support only add and remove), I think it's just noise.

Reply:
@JakeSCahill I believe this is the only outstanding topic. After that, I'll give a green tick.

Reply:
Done 😄